dotcom-rendering/src/model/enhanceTreats.ts

```diff
@@ -0,0 +1,7 @@
+import type { FEFrontCard, TreatType } from '../types/front';
+
+export const enhanceTreats = (treats: FEFrontCard[]): TreatType[] =>
```

Comment:
nit: most other enhance methods return a `DCRType`. Should we stick to that convention?

Reply:
There's no frontend version of a treat. It's just a card (FECardType) so I figured there was no need for the DCR prefix. We also use this type outside of 'fronts' so that felt like another reason to keep it simple

Reply:
I like the simplicity.

Do you think it's ever worth using the `DCR` prefix?

Reply:
I think the prefixes make sense when you have two types that refer to the same thing, the prefix lets you know which is the FE version and which is the DCR version.